src/openms/include/OpenMS/ANALYSIS/ID/SimpleSearchEngineAlgorithm.h

```diff
@@ -70,6 +70,14 @@
 #include <map>
 #include <algorithm>
 
+#ifdef _OPENMP
+  #include <omp.h>
+  #define NUMBER_OF_THREADS (omp_get_num_threads())
+#else
+  #define NUMBER_OF_THREADS (1)
+#endif
```

Comment:
move to .cpp file? otherwise it gets included in every other downstream file...

Reply:
@timosachsenberg Do you really need this here?

Reply:
no I think it can be removed